.github/workflows/go.yml

```diff
@@ -13,9 +13,9 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go: ["1.17.x", "1.18.x"]
+        go: ["1.17.x", "1.18.x", "1.19.x"]
```

Comment:
should 1.17.x be dropped?

Reply:
We haven't been dropping 1.17 from any other package just yet because 1.19 is super new and there's a good chance that many users haven't migrated to 1.18 just yet.

But yes, we should drop this in a future change.